data/supporters.json

```diff
@@ -2368,6 +2368,17 @@
           "twitter": "bjschrijver"
         }
       ]
-    }
+    },
+    {
+      "name": "Bsides Newcastle",
+      "city": "Newcastle Upon Tyne",
+      "country": "United Kingdom",
+      "link": "https://https://bsidesnewcastle.org",
+      "twitter": "bsidesncl",
+      "contacts": [
+        {
+          "name": "Ryan",
+          "email": "ryan@bsidesnewcastle.org, info@bsidesnewcastle.org",
+          "twitter": "chick3nhawk01"
```

Comment:
it looks like it’s missing a closing `}` for the first contact and a closing `]` for the contacts array 🤔